ci/terraform/oidc/api-gateway-frontend.tf

```diff
@@ -77,7 +77,9 @@ resource "aws_api_gateway_deployment" "frontend_deployment" {
       module.doc-app-authorize.method_trigger_value,
       module.orch_auth_code.integration_trigger_value,
       module.orch_auth_code.method_trigger_value,
-      local.account_modifiers_encryption_policy_arn
```

Comment:
Can you make sure you're adding all new policies to the bottom of each list please. See Dan's thread from yesterday in the auth channel

Reply:
Ooh yes, thanks